SPID/include/ExclusiveGroups.h

```diff
@@ -0,0 +1,48 @@
+#pragma once
+#include "LookupConfigs.h"
+
+namespace ExclusiveGroups
+{
+	using GroupName = std::string;
+	using LinkedGroups = std::unordered_map<RE::TESForm*, std::unordered_set<GroupName>>;
```

Comment:
Rename to `FormGroupsMap` and `GroupFormsMap` respectively to avoid usage of `Linked` term to not confuse with `LinkedForms`